lib/routes/helper.js

```diff
@@ -48,6 +48,15 @@ helper.verifyRequest = function(request, resourceConfig, res, handlerRequest, ca
     });
   }
 
+  if (!resourceConfig.handlers.ready) {
+    return helper.handleError(request, res, {
+      status: "503",
+      code: "EUNAVAILABLE",
+      title: "Resource not available",
```

Comment:
Is this the right title here? Could it be more like `Resource temporarily unavailable` - something more fitting to the detail provided below?